source/_docs/http-to-https.md

```diff
@@ -114,7 +114,11 @@ Use the following techniques to replace insecure references to your domain in th
 Clear Drupal and WordPress object caches in the [database and/or in Redis](/docs/drupal-redis/#clear-cache) in addition to manually flush edge caches by going to your Pantheon Dashboard and clicking the **Clear Caches** button.
 
 At this point, all visitors to the site should be able to securely access all pages over HTTPS with no browser warnings.
+
 ## Redirect to HTTPS and the primary domain
+
+As part of best security practices, we suggest you [Require HTTPS with the HSTS Header](/docs/hsts), instead of using a redirect.
+
 Configure redirects to the primary domain with HTTPS in `settings.php` or `wp-config.php` if a visitor arrives either (1) on a different domain or (2) without HTTPS.
```

Comment:
This should be elaborated on. IE what changes in wp-config.php should be made.

Reply:
@Shelob9 do you feel that the linked guide doesn't properly cover how to enable HSTS?

Reply:
I do not.

Pantheon's standard wp-config.php includes a 301 redirect for insecure requests. I am confused if you are recommending that after installing that plugin you recommend I remove the 301 redirect, change it to a 307 or what. Specific changes to wp-config.php should be detailed here.

This is contained in the phrase `Configure redirects` I suggest explaining how to configure a redirect.